packages/core/content-type-builder/admin/src/components/FormModalSubHeader/index.js

```diff
@@ -36,7 +32,9 @@ const FormModalSubHeader = ({
           defaultMessage: 'Add new field',
         },
         {
-          type: upperFirst(formatMessage(intlLabel)),
+          type: upperFirst(
+            formatMessage(customField?.intlLabel ?? { id: getTrad(`attribute.${attributeType}`) })
```

Comment:
In theory I don't think this would be needed since we [throw](https://github.com/strapi/strapi/blob/features/custom-fields/packages/core/admin/admin/src/core/apis/CustomFields.js#L21) when intlLabel is not provided

Reply:
We need to return the attribute translation when the modal is not for a custom field. So I think the ternary I had before is probably more readable.
 
```
const intlLabel = modalType === 'customField' ? customField.intlLabel : { id: getTrad(`attribute.${attributeType}`) }
```

Reply:
I misunderstood what the condition was for sorry